rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -20,8 +20,8 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
 
   @Override
   List<String> buildPackerCommand(String baseCommand, Map<String, String> parameterMap, String absoluteTemplateFilePath) {
-    def packerCommand = ["packer", "build", "-color=false"]
-
+    def packerCommand = [baseCommand, "packer", "build", "-color=false"]
+    packerCommand.removeAll([""])
```

Comment:
The extraneous `""` likely had no ill-effect because the tokenized command just gets space-joined here: https://github.com/spinnaker/rush/blob/master/src/main/groovy/com/netflix/spinnaker/rush/local/scripts/ScriptExecutorLocal.groovy#L58

(I'm thinking maybe you spotted the extra `""` when writing your unit tests and it looked a bit strange. If you think it reads better to strip it out here, I'm ok leaving it as you have it. Maybe also handle null here and in the unit tests, just in case any future added provider passes in null?)